modules/run-consul/README.md

```diff
@@ -193,6 +199,8 @@ configuration file (e.g. `encryption.json`) in the Consul config dir (default lo
 }
 ```
 
+Alternatively, you can run `run-consul` with `--enable-gossip-encryption` and `--gossip-encryption-key` to have this script
+automatically generate the gossip encryption settings in `/opt/consul/config/gossip-encryption.json`.
```

Comment:
I'd actually list this option as the primary way to do it, and make the more manual approach above the "alternative". Same for RPC encryption.